PayPal/src/main/java/com/braintreepayments/api/paypal/AmountBreakdown.kt

```diff
@@ -41,4 +42,17 @@ data class AmountBreakdown(
     val insuranceTotal: String?,
     val shippingDiscount: String?,
     val discountTotal: String?
-) : Parcelable
+) : Parcelable {
+
+    fun toJson(): JSONObject {
+        return JSONObject().apply {
+            put("item_total", itemTotal)
+            put("shipping", shippingTotal)
+            put("handling", handlingTotal)
+            put("tax_total", taxTotal)
+            put("insurance", insuranceTotal)
+            put("shipping_discount", shippingDiscount)
+            put("discount", discountTotal)
```

Comment:
Can we pull the string values of these out into a companion object to match other request param objects?